### ISSUE TITLE
Team settings does not follow to new team

### ISSUE THREAD

**Issue by ChristianJBergstrom** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#8265]
[Hi, I've read the previous feedback posts on this issue and agree that it should be made more clear that when using Teams admin center with templates all settings will not follow.

The table in the below link is what I'm referring to and it makes one think that using TAC with templates you can benefit by using for ex. the team settings from an already existing team ('create from existing team'), which simply doesn't work.

https://docs.microsoft.com/en-us/microsoftteams/get-started-with-teams-templates-in-the-admin-console#team-template-capabilities ]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ef94e9a3-a0a5-11da-f27b-627ac4fa6186
* Version Independent ID: e49db873-ed77-de53-e4dd-be4fc571d734
* Content: [Get started with team templates in the Teams admin center - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/get-started-with-teams-templates-in-the-admin-console#team-template-capabilities)
* Content Source: [Teams/get-started-with-teams-templates-in-the-admin-console.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/get-started-with-teams-templates-in-the-admin-console.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanachin**

**Comment by get-itips** [comment]
Hello @ChristianJBergstrom how are you?

I am looking into this.

**Comment by get-itips** [comment]
Hello @ChristianJBergstrom according to the _[documentation](https://docs.microsoft.com/en-us/microsoftteams/create-template-from-existing-team)_, _Create a team template from an existing team in Microsoft Teams_ is still a _preview or early release feature_ so this might be expected but I'll try to confirm if that is still valid.

**Comment by ChristianJBergstrom** [comment]
@get-itips Hello, let me rephrase. The initial post was simply an example, my apologies. It should be made clearer such as "if using TAC your configuration options are _very_ limited, use PowerShell or Microsoft Graph for team properties settings." Or something like that. Thanks and Merry Christmas!

**Comment by get-itips** [comment]
@ChristianJBergstrom  Do you mean adding a suggestion to the article that, to customize more team settings use for example PowerShell cmdlets?
Merry Christmas for you too!

**Comment by ChristianJBergstrom** [comment]
@get-itips Thanks! Yes, I believe that would be valuable information. Try adding/configuring a template in TAC and you'll understand I'm sure :) Graph or PowerShell should be mentioned as the current way of configuring settings, not sure where though, so will leave that up to you.

**Comment by get-itips** [comment]
Hello @ChristianJBergstrom how are you?
I've done templating in TAC in the past, I am not sure about adding a recommendation to configure more team settings using Ps to this article, I believe that, Administrators that use templates already know the existence of the other tools/procedures to configure teams settings (being in the TAC and also in Ps).

Please let me know if I understood your correctly,

Andrés

**Comment by ChristianJBergstrom** [comment]
@get-itips Hello! To be honest and from my point of view I don't feel it's up to us assuming that administrators already know that using TAC for templates configuration will be a disappointment. It really should be made clear in some context that you simply cannot do any proper configuration in templates using Teams admin center.

**Comment by HowlinWolf-92** [comment]
I agree with Andrés that administrators who use templates already know about the other tools and procedures to configure Teams settings. Therefore, I don't think we should recommend that this article should say that TAC is not proper to handle template settings and thus PowerShell should be used to further customize them.

**Comment by ChristianJBergstrom** [comment]
Let’s agree to disagree. At least I’m not alone when looking at the ”view all page feedback”.

**Comment by get-itips** [comment]
@scanum
